wireshark.sls

```diff
@@ -1,15 +1,48 @@
+# Wireshark is a program used for packet sniffing and analysis.
+# See https://www.wireshark.org/
+# IMPORTANT: Wireshark requires the WinPCAP driver for 2.x and NPCAP driver for 3.x to be installed. This driver an NOT be installed automatically except for the paid OEM/enterprise version. This installation routine only installs Wireshar, not the required driver (!)
+# Npcap: https://nmap.org/npcap/
+
+{% set arch = 'win64' if grains['cpuarch'] == 'AMD64' else 'win32' %}
+{% set source_path = 'https://www.wireshark.org/download/' + arch + '/all-versions' %}
+{% set msiVersions = [
+  '2.4.0', '2.4.1', '2.4.2', '2.4.3', '2.4.4', '2.4.5', '2.4.6', '2.4.7', '2.4.8', '2.4.9', '2.4.10', '2.4.11', '2.4.12', '2.4.13', '2.4.14', '2.4.15', '2.4.16',
+  '2.5.0', '2.5.1',
+  '2.6.0', '2.6.1', '2.6.2', '2.6.3', '2.6.4', '2.6.5', '2.6.6', '2.6.7', '2.6.8', '2.6.9', '2.6.10', '2.6.11', '2.6.12', '2.6.13', '2.6.14', '2.6.15',
```

Comment:
Order is usually newest to oldest, could you reorder these? Also comma isn't needed after last element, not sure if this will cause an issue

Reply:
done 😄 . I have also removed the trailing comma. In my tests the trailing comma did not cause an issue, python / jinja seems more tolerant than e.g. json :)